Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -303,6 +302,7 @@ class PlaceCarouselViewController: UIViewController {
         alertController.addAction(settingsAction)
         alertController.addAction(quitAction)
 
+        AppState.requestPermissions()
```

Comment:
Removed permissions sessions entirely in favor of recording dialog-based events.